contracts/GNUmakefile

```diff
@@ -37,9 +37,12 @@ abigen: ## Build & install abigen.
 mockery: $(mockery) ## Install mockery.
 	go install github.com/vektra/mockery/v2@v2.28.1
 
+.PHONY: foundry
+foundry: ## Install foundry.
+	foundryup --version nightly-5cfed63071ef1bb06f90190d2f678f546bfc971c
+
 .PHONY: foundry-refresh
-foundry-refresh:
-	foundryup
+foundry-refresh: foundry
```

Comment:
is this command even necessary anymore?

Reply:
Yes, helped someone today. It doesn't just update Foundry, it fixes dependency issues when you switch between branches that have different Foundry dependencies. A must have for when we bump the std-lib the next time

Reply:
oh I just meant since `foundry-refresh` just calls `foundry` now - maybe we could avoid the duplication :)